src/ViewModels/WebMapServiceItemViewModel.js

```diff
@@ -108,8 +108,8 @@ var WebMapServiceItemViewModel = function(application) {
     delete this.__knockoutObservables.dataUrlType;
     knockout.defineProperty(this, 'dataUrlType', {
         get : function() {
-            if (defined(this._dataUrlType)) {
-                return this._dataUrlType;
+            if (defined(this._dataUrl)) {
+                return 'direct';
```

Comment:
Roger.  Again, I'm ok with removing functionality we don't currently need, but we should be consistent about it.  ArcGIS catalog items should basically work the same way as WMS catalog items.